helix-core/src/main/java/org/apache/helix/controller/stages/MessageGenerationPhase.java

```diff
@@ -437,10 +454,10 @@ private Message createStateTransitionCancellationMessage(HelixManager manager, R
       boolean isCancellationEnabled, String currentState) {
 
     if (isCancellationEnabled && cancellationMessage == null) {
-      LogUtil.logInfo(logger, _eventId,
-          "Send cancellation message of the state transition for " + resource.getResourceName()
-              + "." + partitionName + " on " + instanceName + ", currentState: " + currentState
-              + ", nextState: " + (nextState == null ? "N/A" : nextState));
+      logger.info("Event {} : Send cancellation message of the state transition for {}.{} on {}, "
```

Comment:
1. I'd like to add the `toState` field to the log.
2. parameterized logging is better than `String.format` in the LogUtil. When error logging level is enabled, the string won't be created in parameterized logging, but it will be created in the LogUtil.
3. The log format is not changed. I don't think someone will really parse the log in the script.

Reply:
Ok.